### ISSUE TITLE
can't handle same headers in different order

### ISSUE THREAD

**Issue by seamusabshere** [faradayio/catcsv#1]
```
$ cat a.csv
city,state
burlington,vt

$ cat b.csv
state,city
wi,madison

$ catcsv a.csv b.csv
city,state
burlington,vt
Error: error reading 'b.csv
Caused by: CSV headers are different from the first file's
```

**Comment by emk** [comment]
Yes, this is definitely not supported at the moment, by design. If this comes up on a regular basis, we could definitely add a mode where we:

1. Parse the second and later CSVs if the columns don't match the first.
2. Shuffle the columns around.
3. Re-serialize the CSVs.

This might also find its way into `dbcrossbar` at some point, which will eventually need a mode where it parses the CSV file and manipulates it while piping it through.
